### ISSUE TITLE
Memory limit reached when trying to count Eloquent models.

### ISSUE THREAD

**Issue by mpociot** [nunomaduro/larastan#33]
When trying to count an Eloquent model using the query builder, I receive a memory limit error:

```php
<?php

namespace App;

use App\User;

class Test
{
    public function foo()
    {
        // Any of these combinations results in a memory limit error
        User::where('name', 'bar')->count();
        User::whereName('bar')->count();
        User::count();

        // It also has nothing to do with facades
        $user = new User();
        $user->count();
    }
}
```

**Comment by nunomaduro** [comment]
Hey @mpociot 👋

Thanks for reporting this false positive.

Your `App\User::class` is normal Laravel Fresh User Model? Or it haves something else? Annotations, etc?

Do you mind of sharing the code present in your user model?

**Comment by nunomaduro** [comment]
I was able to reproduce your problem after installing Barryvdh\LaravelIdeHelper. I imagine that you have that package installed.

**Comment by nunomaduro** [comment]
Hey @mpociot 👋

The issue was already addressed on the version 0.1.1. Do you mind of performing a `composer update nunomaduro/larastan` and check if everything is fine now? ☺️

**Comment by nunomaduro** [comment]
This issue should be fixed by now. Thanks!
